### PR TITLE
chore: add package.json entry options supported

### LINKED ISSUE #1316: Doc request: Reference for all options that can be set in package.json
We should have an overview of all the options that can be configured in `package.json` files under the `prisma` option.

### PR DESCRIPTION
**Pull request by ruheni**
Closes #1316

### CODE REVIEW

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 1]
```diff
@@ -1285,3 +1285,37 @@ prisma studio --browser firefox
...
+The path to the file or script to populate a database with sample database is specified in the `prisma.seed` entry in the `package.json` file. The file specified is used when `prisma db seed` is invoked. Both absolute and relative paths are supported.
+
+See [Seeding your database](/guides/migrate/seed-database) <span class="guide"></span>
+
+```json file="package.json"
+{
+  "name": "my-project",
+  "version": "1.0.0",
+  "prisma": {
+    "seed": "node prisma/seed.js"
```
Comment (millsp) [inline content/400-reference/200-api-reference/200-command-reference.mdx]: Just for consistency with above

```suggestion
    "seed": "node ./prisma/seed.js"
```

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 2]
```diff
@@ -1285,3 +1285,37 @@ prisma studio --browser firefox
...
+  }
+}
+```
```
Comment (millsp) [inline content/400-reference/200-api-reference/200-command-reference.mdx:1321]: Since when is this one available?

Comment (janpio) [reply content/400-reference/200-api-reference/200-command-reference.mdx:1321]: I think we built that with the feature back in the day. @Jolg42 will know better next week.

Comment (ruheni) [reply content/400-reference/200-api-reference/200-command-reference.mdx:1321]: Found it. We released it in 3.0.1

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 3]
```diff
@@ -1285,3 +1285,37 @@ prisma studio --browser firefox
...
+
+```json file="package.json"
+{
+  "name": "my-project",
+  "version": "1.0.0",
+  "prisma": {
+    "seed": "node ./prisma/seed.js"
+  }
+}
+```
```
Comment (ruheni) [inline content/400-reference/200-api-reference/200-command-reference.mdx:1321]:
```suggestion
```

This is available from version 3.0.1 and later.
```

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 4]
```diff
@@ -1285,3 +1285,39 @@ prisma studio --browser firefox
...
+    "schema": "./custom-path-to-schema/schema.prisma"
+  }
+}
+```
+
+This is available from version 2.7.0 and later.
+
+#### <inlinecode>seed</inlinecode> option
+
+The path to the file or script to populate a database with sample database is specified in the `prisma.seed` entry in the `package.json` file. The file specified is used when `prisma db seed` is invoked. Both absolute and relative paths are supported.
```
Comment (Jolg42) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
The command used to populate the datasource is specified in the `prisma.seed` entry in the `package.json` file. It is used when `prisma db seed` is invoked or triggered.
```

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 5]
```diff
@@ -1285,3 +1285,39 @@ prisma studio --browser firefox
...
+  "version": "1.0.0",
+  "prisma": {
+    "schema": "./custom-path-to-schema/schema.prisma"
+  }
+}
+```
+
+This is available from version 2.7.0 and later.
+
+#### <inlinecode>seed</inlinecode> option
```
Comment (Jolg42) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
#### <inlinecode>seed</inlinecode> command
```

Comment (janpio) [reply content/400-reference/200-api-reference/200-command-reference.mdx]: I think this refers to `prisma.seed` "option" (?) in `package.json`, not the `db seed` command?

Comment (Jolg42) [reply content/400-reference/200-api-reference/200-command-reference.mdx]: for the previous one it's "<inlinecode>schema</inlinecode> path" so 🤷🏼

Comment (ruheni) [reply content/400-reference/200-api-reference/200-command-reference.mdx]: I've removed the schema and option keywords.

**Pull request merged**